### PR TITLE
variables field on runs api

### PR DESCRIPTION
**Pull request by brandonc**
## Labels

- [ ] inaccuracy
- [ ] clarification
- [ ] new docs
- [ ] cosmetic bug - fixing broken text or markup
- [ ] enhancement - changing the website's behavior/layout
- [x] api - requires an update to the [changelog](https://www.terraform.io/docs/cloud/api/changelog.html)

### CODE REVIEW

File: content/source/docs/cloud/api/run.html.md [hunk 1]
```diff
@@ -69,6 +69,19 @@ A run performs a plan and apply, using a configuration version and the workspace
 
 Creating a run requires permission to queue plans for the specified workspace. ([More about permissions.](/docs/cloud/users-teams-organizations/permissions.html))
 
+When creating a run, you may optionally provide a list of variable objects containing key and value attributes. These values claim a high priority relative to variables defined at the workspace level. See [Variables Precedence](https://www.terraform.io/docs/cloud/workspaces/variables.html#precedence) section for more information. All values must be expressed as an HCL literal in the same syntax you would use when writing terraform code. See [Terraform types](https://www.terraform.io/docs/language/expressions/types.html#types) for more details.
```
Comment (laurapacilio) [inline content/source/docs/cloud/api/run.html.md:72]:
```suggestion
When creating a run, you may optionally provide a list of variable objects containing key and value attributes. These values apply to that run specifically and take precedence over variables with the same key applied to the workspace(e.g., variable sets). Refer to [Variable Precedence](https://www.terraform.io/docs/cloud/workspaces/variables.html#precedence) for more information. All values must be expressed as an HCL literal in the same syntax you would use when writing Terraform code. Refer to [Types](https://www.terraform.io/docs/language/expressions/types.html#types) in the Terraform documentation for more details.
```

**Comment by brandonc** [comment]
Screen Shot 2021-12-09 at 2 00 34 PM

**Comment by brandonc** [comment]
Screenshot 2021-12-09 at 14-01-11 Runs - API Docs - Terraform Cloud and Terraform Enterprise - Terraform by HashiCorp

File: content/source/docs/cloud/api/run.html.md [hunk 2]
```diff
@@ -79,20 +92,21 @@ This POST endpoint requires a JSON object with the following properties as a req
...
-`data.relationships.configuration-version.data.id` | string | (nothing) | Specifies the configuration version to use for this run. If the `configuration-version` object is omitted, the run will be created using the workspace's latest configuration version.
+Key path                                           | Type                | Default                                            | Description
+---------------------------------------------------|---------------------|----------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+`data.attributes.auto-apply`                       | bool                | (The current Auto Apply option of the workspace)   | Whether to automatically apply changes when a Terraform plan is successful. Defaults to the [workspace Auto Apply](https://www.terraform.io/docs/cloud/workspaces/settings.html#auto-apply-and-manual-apply) setting.
```
Comment (laurapacilio) [inline content/source/docs/cloud/api/run.html.md:97]:
```suggestion
`data.attributes.auto-apply`                       | bool                | The current Auto Apply setting in the workspace  | Whether to automatically apply changes when a Terraform plan is successful. Defaults to the [Auto Apply](/docs/cloud/workspaces/settings.html#auto-apply-and-manual-apply) workspace setting.
```

File: content/source/docs/cloud/api/run.html.md [hunk 3]
```diff
@@ -79,20 +92,21 @@ This POST endpoint requires a JSON object with the following properties as a req
...
+`data.attributes.is-destroy`                       | bool                | false                                              | Specifies if this plan is a destroy plan, which will destroy all provisioned resources. Mutually exclusive with `refresh-only`.
```
Comment (laurapacilio) [inline content/source/docs/cloud/api/run.html.md:98]:
```suggestion
`data.attributes.is-destroy`                       | bool                | false                                              | Specifies whether this plan is a destroy plan that will destroy all provisioned resources. Mutually exclusive with `refresh-only`.
```

File: content/source/docs/cloud/api/run.html.md [hunk 4]
```diff
@@ -79,20 +92,21 @@ This POST endpoint requires a JSON object with the following properties as a req
...
+`data.attributes.message`                          | string              | "Queued manually via the Terraform Enterprise API" | Specifies the message to be associated with this run.
+`data.attributes.refresh`                          | bool                | true                                               | Specifies whether or not to refresh the state before a plan.
+`data.attributes.refresh-only`                     | bool                | false                                              | Whether this run should use the refresh-only plan mode, which will refresh the state without modifying any resources. Mutually exclusive with `is-destroy`.
```
Comment (laurapacilio) [inline content/source/docs/cloud/api/run.html.md:101]:
```suggestion
`data.attributes.refresh-only`                     | bool                | false                                              | Specifies whether this run should use the refresh-only plan mode, which will refresh the state without modifying any resources. Mutually exclusive with `is-destroy`.
```

File: content/source/docs/cloud/api/run.html.md [hunk 5]
```diff
@@ -79,20 +92,21 @@ This POST endpoint requires a JSON object with the following properties as a req
...
+`data.attributes.replace-addrs`                    | array[string]       | (nothing)                                          | Specifies an optional list of resource addresses to be passed to the `-replace` flag.
+`data.attributes.target-addrs`                     | array[string]       | (nothing)                                          | Specifies an optional list of resource addresses to be passed to the `-target` flag.
+`data.attributes.variables`                        | array[{key, value}] | empty                                              | Specifies an optional list of run-specific variable values. See [Run-Specific Variables](https://www.terraform.io/docs/cloud/workspaces/managing-variables.html#run-specific-variables) in the Terraform Cloud workspaces documentation for more information.
```
Comment (laurapacilio) [inline content/source/docs/cloud/api/run.html.md:104]:
```suggestion
`data.attributes.variables`                        | array[{key, value}] | empty                                              | Specifies an optional list of run-specific variable values. Refer to [Run-Specific Variables](https://www.terraform.io/docs/cloud/workspaces/managing-variables.html#run-specific-variables) in the Terraform Cloud documentation for more information.
```

File: content/source/docs/cloud/api/run.html.md [hunk 6]
```diff
@@ -79,20 +92,21 @@ This POST endpoint requires a JSON object with the following properties as a req
...
+`data.relationships.workspace.data.id`             | string              | (nothing)                                          | Specifies the workspace ID where the run will be executed.
+`data.relationships.configuration-version.data.id` | string              | (nothing)                                          | Specifies the configuration version to use for this run. If the `configuration-version` object is omitted, the run will be created using the workspace's latest configuration version.
```
Comment (laurapacilio) [inline content/source/docs/cloud/api/run.html.md:106]: Is (nothing) a convention we usually use? If so I accept it. If not, I wonder is there a convention we typically use for no default. Or, should we just leave that cell blank?

**Review by laurapacilio — APPROVED** [review]
Couple nits for you - thank you! Feel free to merge ASAP before the docs freeze!

**Pull request merged**